pkg/logfilter/logfilter.go

```diff
@@ -14,9 +15,10 @@ import (
 )
 
 const (
-	defaultEnhancedFilterTime = "30s"
-	ptp4lProcessName          = "ptp4l"
-	phc2sysProcessName        = "phc2sys"
+	defaultEnhancedFilterTime      = "30s"
+	defaultEnhancedFilterThreshold = "0"
```

Comment:
isn';t this more of a offset related ?
could you name it as offsetFlushThreshold

Reply:
okay I can rename it